### PR TITLE
Add module auth_signup_verify_email.

### PR DESCRIPTION
**Pull request by yajo**
Relaunch https://github.com/OCA/website/pull/94 for the right repo. No other changes. Sorry for the inconvenience.

CC @rafaelbn @dreispt @sergio-incaser @StefanRijnhart.

### CODE REVIEW

File: auth_signup_verify_email/controllers/main.py [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+                not http.request.params.get("password")):
+            return self.passwordless_signup(http.request.params)
+        else:
+            return super(SignupVerifyEmail, self).web_auth_signup(*args, **kw)
+
+    def passwordless_signup(self, values):
+        qcontext = self.get_auth_signup_qcontext()
+
+        # Check good format of e-mail
+        from validate_email import validate_email
```
Comment (moylop260) [inline auth_signup_verify_email/controllers/main.py]: Could you check next guideline https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#importerror please?

And next one: https://github.com/OCA/maintainer-tools/blob/master/CONTRIBUTING.md#imports

**Comment by rafaelbn** [comment]
From :https://github.com/OCA/website/pull/94

> Let's ask @gurneyalex  to add the validate_email python library to the runbot configuration

File: auth_signup_verify_email/README.rst [hunk 2]
```diff
@@ -0,0 +1,84 @@
...
+=====
+
+To use this module, you need to:
+
+* Log out.
+* `Sign up </web/signup>`_ with a valid email.
+
+.. image:: https://odoo-community.org/website/image/ir.attachment/5784_f2813bd/datas
+   :alt: Try me on Runbot
+   :target: https://runbot.odoo-community.org/runbot/186/8.0
```
Comment (StefanRijnhart) [inline auth_signup_verify_email/README.rst]: This is now repo 149 ;-) https://github.com/OCA/maintainer-tools/blob/master/tools/repos_with_ids.txt#L28

File: auth_signup_verify_email/README.rst [hunk 3]
```diff
@@ -0,0 +1,84 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+   :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+   :alt: License: AGPL-3
+
+======================
+Verify email at signup
+======================
+
+This module was written to extend the functionality of public sign up to force
```
Comment (elicoidal) [inline auth_signup_verify_email/README.rst]: This module extends the functionality of public sign up and forces

File: auth_signup_verify_email/README.rst [hunk 4]
```diff
@@ -0,0 +1,84 @@
...
+users to provide a valid email address.
+
+To reinforce this requirement, the user will not need to provide a password at
```
Comment (elicoidal) [inline auth_signup_verify_email/README.rst]: To achieve this requirement, the user does not need to provide a password at

File: auth_signup_verify_email/README.rst (hunk 3 above)
Comment (yajo) [inline auth_signup_verify_email/README.rst]: You forgot a comma.

**Comment by yajo** [comment]
All fixed.

**Comment by rafaelbn** [comment]
Tested :+1: , please @sergio-incaser @carlos-incaser could you review again?
cc @janetrp @dgzurita 100% needed in ecommerce when registering please review

**Comment by rafaelbn** [comment]
@pedrobaeza please pip install validate_email, thanks!

**Comment by pedrobaeza** [comment]
Done and restarted build

**Comment by rafaelbn** [comment]
I tested in runbot and in local, also customer has it in devel enviroment, works fine. In runbot you must quick set and delete smtp server. Module is :+1:

**Comment by moylop260** [comment]
:+1: (without test)

**Comment by carlosdauden** [comment]
:+1:

**Pull request merged**